public/app/features/explore/Logs/Logs.tsx

```diff
@@ -158,6 +167,18 @@ class UnthemedLogs extends PureComponent<Props, State> {
     }
   }
 
+  componentDidUpdate(prevProps: Readonly<Props>): void {
+    if (this.props.loading && !prevProps.loading && this.props.panelState?.logs?.id) {
```

Comment:
Just curious, have you considered other places in code for this update? like `runQueries` in redux? I am not sure if it would better, but I am just wondering.

Reply:
I thought about it and for now this felt sufficient, especially because I don't know how often e.g. `runQueries` runs.